src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -203,749 +203,114 @@ private ImmutableArray<ISwitchCase> GetPatternSwitchStatementCases(BoundPatternS
             });
         }
 
-        private static BinaryOperationKind GetLabelEqualityKind(BoundSwitchLabel label)
+        internal class Helper
```

Comment:
Sure, added the comment to the list of items tracked here: https://github.com/dotnet/roslyn/issues/20204